dali/operators/audio/mfcc/mfcc_test.cc

```diff
@@ -19,36 +19,70 @@ namespace dali {
 namespace detail {
 namespace test {
 
-void check_lifter_coeffs(LifterCoeffs& coeffs, double lifter, int64_t length) {
+void check_lifter_coeffs(span<const float> coeffs, double lifter, int64_t length) {
   ASSERT_EQ(length, coeffs.size());
   auto coeffs_data = coeffs.data();
   ASSERT_NE(nullptr, coeffs_data);
   for (int64_t i = 0; i < length; i++) {
     float expected = 1.0 + 0.5 * lifter * std::sin((i + 1) * M_PI / lifter);
-    EXPECT_FLOAT_EQ(expected, coeffs_data[i]);
+    EXPECT_NEAR(expected, coeffs_data[i], 1e-4);
   }
 }
 
-TEST(LifterCoeffs, correctness) {
-  LifterCoeffs coeffs;
+TEST(LifterCoeffsCPU, correctness) {
+  LifterCoeffs<CPUBackend> coeffs;
 
   auto lifter = 0.0f;
   coeffs.Calculate(10, lifter);
   ASSERT_TRUE(coeffs.empty());
 
   lifter = 1.234f;
   coeffs.Calculate(10, lifter);
-  check_lifter_coeffs(coeffs, lifter, 10);
+  check_lifter_coeffs(span<const float>(coeffs.data(), coeffs.size()), lifter, 10);
```

Comment:
`make_cspan(coeffs)` should work

Reply:
Even `make_span<coeffs>` would - there's an implicit conversion to a span of const-qualified objects.

Reply:
done